.circleci/config.yml

```diff
@@ -146,14 +168,44 @@ jobs:
             - kube-compose-controller-coverage.tar
             - kube-compose-installer.tar
             - kube-compose-e2e-tests.tar
+  e2e-kind:
+    docker: *public-golang
+    working_directory: /root/src/github.com/docker/compose-on-kubernetes
+    environment:
+      GOPATH: "/root"
+    steps:
+      - attach_workspace:
+          at: /root
+      - setup_remote_docker:
+          version: 18.06.0-ce
+      - run:
+          name: "Prepare Kind and run E2E tests"
+          command: |
+            git clone https://github.com/kubernetes-sigs/kind.git /root/src/sigs.k8s.io/kind
```

Comment:
I think this should be done in a `Dockerfile` instead... Too much logic for a CI config file.

Reply:
This is not trivial, as the kind tool create config files on the machine, that we need to pass to other containers etc. Can be done in a followup maybe ?